website/content/ja/wgs/platforms/whitepaper/index.md

```diff
@@ -6,496 +6,261 @@ description: "この論文は、エンタープライズのリーダー、エン
-valuable initially.
-
-Categories of metrics that will help enterprises and platform teams understand
-the impact of their platforms include the following:
-
-### User satisfaction and productivity
-
-The first quality sought by many platforms is to improve user experience in order
-to increase productivity. Metrics that reflect user satisfaction and
-productivity include the following:
-
-- Active users and retention: includes number of capabilities provisioned and user growth/churn
-- "Net Promoter Score" (NPS) or other surveys that measure user satisfaction with a product
-- Metrics for developer productivity such as those discussed in the SPACE framework [[4]]
-
-### Organizational efficiency
-
-Another benefit sought from many platforms is to efficiently provide common
-needs to a large user base. This is often achieved by enabling user self-service
-and reducing manual steps and required human intervention while implementing
-policies to guarantee safety and compliance. To measure the efficiency of a
-platform in reducing common work, consider measures such as these:
-
-- Latency from request to fulfillment of a service or capability, such as a database or test environment
-- Latency to build and deploy a brand new service into production
-- Time for a new user to submit their first code changes to their product
-
-### Product and feature delivery
-
-The ultimate objective of internal platforms is to deliver business value to
-customers faster, so measuring impact on a business's own product and feature
-releases demonstrates that the objectives of the platform are being met. The
-DevOps Research and Assessment (DORA) institute at Google suggests [[5]]
-tracking the following metrics:
-
-- Deployment frequency
-- Lead time for changes
-- Time to restore services after failure
-- Change failure rate
-
-Generally, a key objective of platform teams is to align infrastructure and
-other IT capabilities with an enterprise's value streams - its products. And so
-ultimately the success of an organization's products and applications are the
-true measure of the success of a platform.
-
-## Capabilities of platforms
-
-As we've described, a platform for cloud-native computing offers and composes
-capabilities and services from many supporting providers. These providers may be
-other teams within the same enterprise or third parties like cloud service
-providers. In a nutshell, platforms bridge from underlying _capability
-providers_ to platform users like application developers; and in the process
-implement and enforce desired practices for security, performance, cost
-governance and consistent experience. The following graphic illustrates the
-relationships between products, platforms, and capability providers.
+個々の能力または能力セットに対して一貫性のある、コンプライアンスに準拠した体験を提供することで、内部プラットフォームは最終的に、ユーザーにとって価値のある製品をより簡単に、より効率的に提供することを可能にします。
+
+## プラットフォームの特性
+
+プラットフォームとは何か、そしてなぜ組織がプラットフォームを構築したいと考えるのかを定義したうえで、プラットフォームの成功に影響を与える主な要素をいくつか特定してみましょう。
```

Comment:
I think this sentence is an introduction to the following list of platform attributes and has no definition of what a platform is.
So, it seems more natural if we translate it like the following.

```suggestion
プラットフォームとは何か、そしてなぜ組織がプラットフォームを構築したいと考えるのかを定義しましたので、プラットフォームの成功に影響を与える主な要素をいくつか特定してみましょう。
```